util/utils.js

```
@@ -768,14 +892,50 @@ function runTests(
  * @param shard - the number of shards the test is run for (defaults to 0)
  * @param crudOptions - the crudOptions to be used with the test (see getDefaultCrudOptions() for defaults)
  * @param excludeTestbed - Exclude testbed information from results
+ * @param variantName - the variant name to be set in mongod
+ * @param variants - the variant values
  * @returns {{}} the results of a run set of tests
  */
-function mongoPerfRunTests(
-    threadCounts, multidb, multicoll, seconds, trials, includeFilter, excludeFilter, shard,
-    crudOptions, excludeTestbed, printArgs, shareDataset, mongoeBenchOptions, username, password) {
-    testResults = runTests(
-        threadCounts, multidb, multicoll, seconds, trials, includeFilter, excludeFilter, shard,
-        crudOptions, excludeTestbed, printArgs, shareDataset, mongoeBenchOptions, username, password);
+function mongoPerfRunTests(threadCounts,
+                           multidb,
+                           multicoll,
+                           seconds,
+                           trials,
+                           includeFilter,
+                           excludeFilter,
+                           shard,
+                           crudOptions,
+                           excludeTestbed,
+                           printArgs,
+                           shareDataset,
+                           variantName,
+                           variants,
+                           mongoeBenchOptions,
+                           username,
+                           password) {
+    var testResults = "";
+    var oldValue = db.adminCommand({getParameter: 1, [variantName]: 1});
```

does this work if variantName is not set?

---

Thanks for catching it. It fails silently, but good to prevent it.